doc/source/ray-observability/user-guides/debug-apps/optimize-performance.rst

```diff
@@ -347,6 +347,21 @@ Here are the steps to use PyTorch Profiler during training with Ray Train or bat
 * Visualize the results with tools like Tensorboard.
 
 
+GPU Profiling with Nsight Profiler
+----------------------------------
+Here are the steps to use Nsight Profiler with Ray Core:
+
+* Follow the `Nsight Profiler documentation <https://docs.nvidia.com/nsight-systems/UserGuide/index.html>`_ to specify which APIs to be profiled.
+
+* Setup the Ray tasks or actors which you want to profile by following this :ref:`user guide <ray-profiling>`.
```

Comment:
Can you link to the specific section instead of this page (the section for GPU profiling)